src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.js

```diff
@@ -187,11 +187,19 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
         }
 
         volumeSlider.addEventListener('change', function () {
-
             if (currentPlayer) {
```

Comment:
This should really be a separate function as it's duplicated thrice. Example

```javascript
var newVolume = this.value;
function setVolume() {
    if (currentPlayer) {
        currentPlayer.setVolume(newVolume );
    }
}
volumeSlider.addEventListener('change', setVolume);
volumeSlider.addEventListener('mousemove', setVolume);
volumeSlider.addEventListener('touchmove', setVolume);
```

Reply:
Well, I can redo it like the other 2 files. Was mostly just leaving them as unchanged as feasible.

Reply:
Yes, please make it a separate function.